src/parseNode.js

```diff
@@ -28,6 +28,7 @@ type ParseNodeTypes = {
         type: "array",
```

Comment:
I think it's better to make `ColSeparationType` a type and import it here like `AlignSpec`.